HolidayList/Holiday.cs

```diff
@@ -1,11 +1,404 @@
-﻿// <auto-generated />
-//
-// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
-//
-//    using QuickType;
-//
-//    var welcome = Welcome.FromJson(jsonString);
+﻿//// <auto-generated />
```

Comment:
Don't commit commented out code.   Just delete it.  We can always go back in history in version control to see what used to be there.